lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc

```diff
@@ -213,17 +213,17 @@ function dosomething_campaign_preprocess_action_page(&$vars, &$wrapper) {
   // 1) Materials
 
   // Build the unique share link.
-  $share_path = 'user/' . $vars['user']->uid;
+  $share_path = 'user/' . $vars['user']->uuid;
```

Comment:
@angaither do you happen to know what we would want the link to be if they don't have one?